src/main/java/com/mixpanel/android/util/RemoteService.java

```diff
@@ -10,7 +10,7 @@
 
 
 public interface RemoteService {
-    boolean isOnline(Context context);
+    boolean isOnline(Context context, OfflineMode offlineMode);
```

Comment:
Instead of modifying the interface and force to pass that object every time. Could we do `MPConfig.getInstance(context).getOfflineMode()` inside `HttpService`?